cypress/page/verify-step.page.ts

```diff
@@ -0,0 +1,12 @@
+class VerifyPage {
+    private confirmationText: string;
+
+    constructor() {
+      this.confirmationText = "#center_column > div > p > strong";
+    }
+
+    public verifyPurchase(): void {
+      cy.get(this.confirmationText).should("have.text", "Your order on My Store is complete.");
```

Comment:
```suggestion
      cy.get(this.confirmationText);
```
The verification should be in the test